src/coreComponents/constitutive/fluid/multifluid/CO2Brine/functions/PVTFunctionBase.hpp

```diff
@@ -113,17 +113,6 @@ class PVTFunctionBase
 
   virtual ~PVTFunctionBase() = default;
 
-  using CatalogInterface = dataRepository::CatalogInterface< PVTFunctionBase,
-                                                             string const &,
-                                                             array1d< string > const &,
-                                                             array1d< string > const &,
-                                                             array1d< real64 > const &,
-                                                             bool const >;
-  static typename CatalogInterface::CatalogType & getCatalog()
-  {
-    static CatalogInterface::CatalogType catalog;
-    return catalog;
-  }
```

Comment:
what happened to the catalog? Is it unused...will it remain unused?

Reply:
It was both unused by `PVTFunctionBase ` and `FlashModelBase`

Reply:
It is a proposal as we saw that it was not used anywhere